README.md

```diff
@@ -67,6 +67,7 @@ kubectl get configmap storage-checkup-config -n <target-namespace> -o yaml
 |status.result.pvcBound|PVC of 10Mi created and bound by the provisioner||
 |status.result.storageProfilesWithEmptyClaimPropertySets|StorageProfiles with empty claimPropertySets (unknown provisioners)||
 |status.result.storageProfilesWithSpecClaimPropertySets|StorageProfiles with spec-overriden claimPropertySets||
+|status.result.storageWithSmartClone|Storage with smart clone support (CSI/snapshot)||
```

Comment:
I think this key and probably also storageWithRWX and storageMissingVolumeSnapshotClass names should change to have the prefix of storageProfiles i.e storage*Profiles*WithSmartClone etc.. since you dont actuall check the storage itself.. only what is configured in the storageclass, it is possible that the storage has smartClone capabilities but the storageprofile is not updates with it

Reply:
Actually I don't like the _storageProfiles_ prefix of the previous two result fields either, as names become too long, and should be readable by users. I would prefer aligning all with _storage_ and in the doc details each of them can start with `StorageProfiles with XYZ...`. wdyt?

Reply:
Yeah I get what you say about the too long names, so sure for the variable you can make it shorted but in the description highlight that it talks about the storage profile. And you should make it consistent with every check that relates to the storageprofiles